### PR TITLE
us_la_western

### PR DESCRIPTION
**Pull request by ManuBenito259**
Added the LA Western region and its static feed Sportran

### CODE REVIEW

File: regions/us_la_western.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (nighthawk) [inline regions/us_la_western.json:1]: File name should use `-` not `_`

File: regions/us_la_western.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "name": "Mansfield"
+      },
+      {
+        "lat":  32.868053,
+        "lng": -93.986663,
+        "name": "Vivian"
+      }
+    ],
+    "polygon": {
+      "type": "Polygon",
```
Comment (nighthawk) [inline regions/us_la_western.json:31]: This polygon is oddly narrow and very tall. Should be save to extend East to include Natchitoches to allow for car + PT results coming from there.

Screen Shot 2022-09-16 at 11 33 41

File: regions/us_la_western.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  "coverage": {
+    "cities": [
+      {
+      "lat": 32.469508,
+      "lng": -93.756637,
+      "name": "Shreveport"
+      },
+      {
+        "lat": 32.037184,
+        "lng": 93.714752,
```
Comment (fersagui) [inline regions/us_la_western.json:21]: This is wrong

File: regions/us_la_western.json (hunk 2 above)
Comment (fersagui) [reply regions/us_la_western.json:31]: Yes, agree. Let's make this one bigger to cover the surrounding cities.

**Review by fersagui — CHANGES_REQUESTED** [review]

File: regions/us_la_western.json (hunk 3 above)
Comment (fersagui) [inline regions/us_la_western.json:21]:
```suggestion
        "lng": -93.714752,
```

File: regions/us-la-shreveport.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
+{
+  "code": {
+    "countryCode": "US",
+    "subdivisionCode": "LA",
+      "cityName": "Shreveport"
+  },
+  "timezone": "America/New_York",
```
Comment (nighthawk) [inline regions/us-la-shreveport.json:7]: Is this correct? This region is very much in the centre of the US. Is that still NY timezone?

Comment (ManuBenito259) [reply regions/us-la-shreveport.json:7]: Its not, its actually Chicago, fixing it right away

Comment (fersagui) [reply regions/us-la-shreveport.json]: @damianVarela I really like your idea. 

> a region can have only one timezone, right?

A default one must be set. The idea is to creation which are always in the same timezone. Only the super region could have more.

File: regions/us-la-shreveport.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  "locale": {
+    "language": "en",
+    "country": "US"
+  },
+  "coverage": {
+    "cities": [
+      {
+      "lat": 32.469508,
+      "lng": -93.756637,
+      "name": "Shreveport"
```
Comment (fersagui) [inline regions/us-la-shreveport.json:17]: The full name with state and country is "Shreveport, LA, US". 

@nighthawk make sense to use simple city names here or the complete ones are mandatory for the geocoder?

Comment (nighthawk) [reply regions/us-la-shreveport.json:17]: It should use the simple name here. The full one can be built using what's in `code` at the top level.

**Review by fersagui — APPROVED** [review]

**Pull request merged**